app-backend/build.sbt

```diff
@@ -175,6 +175,10 @@ lazy val batch = Project("batch", file("batch"))
       Dependencies.scopt
     )
   })
+  .settings(assemblyShadeRules in assembly := Seq(
+      ShadeRule.rename("shapeless.**" -> "shadeshapeless.@1").inAll
```

Comment:
Let's call this shaded package somehow like `com.azavea.shaded` just in case we'll have more things to shade.